lib/player.rb

```diff
@@ -0,0 +1,12 @@
+class Player
+  attr_reader :token
+
+  def initialize(token)
+    @token = token
+  end
+
+  def token
+    @token.freeze
```

Comment:
If you want to override the `token` method, you don't need the `attr_reader`. It's not doing anything here.